rasa/core/channels/botframework.py

```diff
@@ -142,7 +142,7 @@ def prepare_message(
     async def send_elements(
         self, recipient_id: Text, elements: Iterable[Dict[Text, Any]], **kwargs: Any
     ) -> None:
-        for e in elements:
+        for e in elements[0]:
```

Comment:
```suggestion
        if not elements:
            return
        for e in elements[0]:
```

Reply:
this is a bit of an odd piece of code, would be great if you can also add a reference to the botframework connector docs